apps/src/gamelab/animationListModule.js

```diff
@@ -393,8 +394,12 @@ export function editAnimation(key, props) {
  * @returns {function}
  */
 export function deleteAnimation(key) {
-  return dispatch => {
-    dispatch(selectAnimation(null));
+  return (dispatch, getState) => {
+    const orderedKeys = getState().animationList.orderedKeys;
+    const currentSelectionIndex = orderedKeys.indexOf(key);
+    let keyToSelect = (currentSelectionIndex === 0) ? 1: (currentSelectionIndex - 1);
```

Comment:
Interesting subtle behavior matrix here - I didn't actually see a test covering the lower-right case.

**Table of selection behaviors on delete**

|   | Deleting index 0 | Deleting a different index |
|---|---|---|
|Deleting the last item| Select nothing | N/A |
|Not deleting the last item| Select the following index (1) | Select the previous index  |

(also spacing in the ternary: `? 1 : (`)